ironfish-cli/scripts/import-export-test.sh

```diff
@@ -0,0 +1,150 @@
+    if [ $? -ne 0 ]; then
+        echo "Import failed for $ACCOUNT_NAME"
+        exit 1
+    fi
+    check_import_success "$IMPORT_OUTPUT" "$ACCOUNT_NAME"
+    DELETE_OUTPUT=$(yarn --cwd ./ironfish-cli start wallet:delete $ACCOUNT_NAME --wait)
+    # verify return code of delete
+    if [ $? -ne 0 ]; then
+        echo "Deletion failed for $ACCOUNT_NAME"
+        exit 1
+    fi
+    check_delete_success "$DELETE_OUTPUT" "$ACCOUNT_NAME"
+}
+
+
+function import_account_by_pipe() {
+    echo "Testing import by pipe."
+    IMPORT_OUTPUT=$(expect -c "
+        spawn sh -c \"cat $TEST_FILE | yarn --cwd ./ironfish-cli start wallet:import\"
+        expect {
+            \"Enter a new account name:\" {
+                send \"$ACCOUNT_NAME\\r\"
+                exp_continue
+            }
+            \"Account $ACCOUNT_NAME imported\" {
+                set output \$expect_out(buffer)
+                exp_continue
+            }
+            eof {
+                set output \$expect_out(buffer)
+            }
+        }
+        puts \$output
+    ")
+    # verify return code of import
+    if [ $? -ne 0 ]; then
+        echo "Import failed for $ACCOUNT_NAME"
+        exit 1
+    fi
+    check_import_success "$IMPORT_OUTPUT" "$ACCOUNT_NAME"
+    DELETE_OUTPUT=$(yarn --cwd ./ironfish-cli start wallet:delete $ACCOUNT_NAME --wait)
+    # verify return code of delete
+    if [ $? -ne 0 ]; then
+        echo "Deletion failed for $ACCOUNT_NAME"
+        exit 1
+    fi
+    check_delete_success "$DELETE_OUTPUT" "$ACCOUNT_NAME"
+}
+
+
+
+
+function import_account_by_path() {
+    echo "Testing import by path."
+    # the cwd is already ironfish-cli, so we actually need to trim an extra "ironfish-cli" from the path
+    ACCOUNT_BY_PATH_TEST_FILE="./"${TEST_FILE:14}
+    IMPORT_OUTPUT=$(expect -c "
+        spawn yarn --cwd ./ironfish-cli start wallet:import --path $ACCOUNT_BY_PATH_TEST_FILE
+        expect {
+            \"Enter a new account name:\" {
+                send \"$ACCOUNT_NAME\\r\"
+                exp_continue
+            }
+            \"Account $ACCOUNT_NAME imported\" {
+                set output \$expect_out(buffer)
+            }
+            eof {
+                set output \$expect_out(buffer)
+            }
+        }
+        puts \$output
+    ")
+    # verify return code of import
+    if [ $? -ne 0 ]; then
+        echo "Import failed for $ACCOUNT_NAME"
+        exit 1
+    fi
+    check_import_success "$IMPORT_OUTPUT" "$ACCOUNT_NAME"
+    DELETE_OUTPUT=$(yarn --cwd ./ironfish-cli start wallet:delete $ACCOUNT_NAME --wait)
+    # verify return code of delete
+    if [ $? -ne 0 ]; then
+        echo "Deletion failed for $ACCOUNT_NAME"
+        exit 1
+    fi
+    check_delete_success "$DELETE_OUTPUT" "$ACCOUNT_NAME"
+}
+
+#this script is always run from the root of ironfish
+TEST_VECTOR_LOCATION='./ironfish-cli/scripts/import-export-test-vector/'
+FORMAT_ARRAY=( blob json mnemonic)
+for VERSION in {65..72}
```

Comment:
This needs to be updated every release. What if we change this to just loop over every file in the fixtures directory and run an import test on it?